### DISCUSSION TITLE
Azure Linux .NET container images are now publicly supported

### DISCUSSION THREAD

**Discussion by lbussell** [dotnet/dotnet-docker#5022] [Announcements]
.NET container images based on [Azure Linux](https://github.com/microsoft/CBL-Mariner) (formerly CBL-Mariner) are now publicly supported.
It is available in both **distroless** and **non-distroless** versions for .NET 6, .NET 7, and .NET 8.

Azure Linux images are tagged as `cbl-mariner2.0`. For example:
- `mcr.microsoft.com/dotnet/sdk:8.0-cbl-mariner2.0`
- `mcr.microsoft.com/dotnet/aspnet:8.0-cbl-mariner2.0`
- `mcr.microsoft.com/dotnet/runtime:8.0-cbl-mariner2.0`
- `mcr.microsoft.com/dotnet/runtime-deps:8.0-cbl-mariner2.0-distroless`

Full tag listings are available on [MAR](https://mcr.microsoft.com/catalog?search=dotnet/).

**Comment by richlander** [comment]
Support level offered is: "Same as Ubuntu".

Per https://github.com/dotnet/dotnet-docker/issues/5004

**Comment by sonnyhcl** [comment]
@lbussell What's release schedule for these Azure Linux images? Is it documented anywhere?

**Reply by MichaelSimons** [reply]
The .NET image update policy is documented [here](https://github.com/dotnet/dotnet-docker/blob/main/README.md#image-update-policy)
